modules/cnf-working-with-fast-event-notifications-api.adoc

```diff
@@ -0,0 +1,283 @@
+// Module included in the following assemblies:
+//
+// * networking/configuring-ptp.adoc
+
+[id="cnf-working-with-fast-event-notifications-api_{context}"]
+= Working with the PTP fast event notifications REST API
+
+You can use the PTP fast event notifications REST API to subscribe an application to the PTP events that are generated for the parent node. PTP fast events notifications are available on each node where a PTP capable network interface is configured.
+
+Events are available from the REST API at `/cluster/node/<node_name>/ptp`, where `<node_name>` is the cluster node running the DU application.
```

Comment:
Events are available to subscribe using this  resource address `/cluster/node/<node_name>/ptp`